src/api/app/models/staging/workflow.rb

```diff
@@ -101,11 +114,11 @@ def update_staging_projects_managers_group
     # FIXME: This assignation is need because after store a staging_project
     # the object is reloaded and we lost the changes.
     self.managers_group = new_managers_group
-  end
 
-  def add_reviewer_group
-    role = Role.find_by_title('reviewer')
-    project.relationships.find_or_create_by(group: managers_group, role: role)
+    # update reviewer group in staging project
+    reviewer_role = Role.find_by_title('reviewer')
+    relationship = project.relationships.find_by(group: old_managers_group, role: reviewer_role)
+    relationship.update(group: new_managers_group)
```

Comment:
Question: what happens if that group already exists? I see an uniqueness validation in relationships on role and group. I assume the update would fail and the old group would stay?

Reply:
yes, the old group would stay. I guess we need to accept the review if there is already one for that group :thinking:

Reply:
@coolo, @Ana06, modified. Please have a look again.